src/relier/OAuthUtils.js

```diff
@@ -83,7 +86,7 @@ class OAuthUtils {
 
       queryParams.keys_jwk = base64JwkPublicKey; // eslint-disable-line camelcase
 
-      const authUrl = `${this.oauthServer}/authorization` + util.objectToQueryString(queryParams);
+      const authUrl = `${this.contentServer}/authorization` + util.objectToQueryString(queryParams);
```

Comment:
@rfk do we want to make a request to https://accounts.firefox.com/.well-known/openid-configuration during these requests and get the endpoints and only fallback to `const` values? or something different?

Reply:
I think we can grapple with that in a follow-up, a `const` will be simplest for now.